src/main/java/com/glencoesoftware/omero/ms/image/region/ImageRegionMicroserviceVerticle.java

```diff
@@ -234,13 +240,21 @@ private void getMicroserviceDetails(RoutingContext event) {
         String version = Optional.ofNullable(
             this.getClass().getPackage().getImplementationVersion()
         ).orElse("development");
+        int maxTileLength = Integer.parseInt(
+                Optional.ofNullable(
+                    preferences.getProperty("omero.pixeldata.max_tile_length")
+                ).orElse("1024").toLowerCase()
+            );
         JsonObject resData = new JsonObject()
                 .put("provider", "ImageRegionMicroservice")
                 .put("version", version)
                 .put("features", new JsonArray()
-                                     .add("flip")
-                                     .add("mask-color")
-                                     .add("png-tiles"));
+                                 .add("flip")
+                                 .add("mask-color")
+                                 .add("png-tiles"))
+                .put("options", new JsonArray()
```

Comment:
Do we need the `JsonArray` wrapper or can we straight go to the `JsonObject` here?

Reply:
Depends how you want to use it I guess. I was trying to keep them ordered but we can just use a dictionary too.

/cc @kkoz

Reply:
A plain dictionary would make it easiest to handle on my end.